src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/android/CommonPage.java

```diff
@@ -49,4 +49,9 @@ public void clickCloseModalButton() {
     public boolean isItemByTextPresent(String text) {
         return itemByText.format(text).isPresent(3);
     }
+
+    @Override
+    public void SwipeToElementByText(String text) {
+        swipe(itemByText.format(text), Direction.VERTICAL, 3, 1000);
```

Comment:
wrong logic. we have to put all the options from More menu into enum and add just 1 method - isMoreOptionPresent, swipe will be inside the method